bears/general/LineCountBear.py

```diff
@@ -10,12 +10,13 @@ class LineCountBear(LocalBear):
     LICENSE = 'AGPL-3.0'
     CAN_DETECT = {'Formatting'}
 
-    def run(self, filename, file, max_lines_per_file: int):
+    def run(self, filename, file, max_lines_per_file: int,
+            min_lines_per_file: int=0):
         """
         Count the number of lines in a file and ensure that they are
         smaller than a given size.
```

Comment:
Since we are adding `min_lines_per_file`, we need to include that in the description  -> `Count the number of lines in a file and ensure that they lie within a range of given sizes`.